src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.css

```diff
@@ -11,7 +11,7 @@
   background: var(--theme-toolbar-background);
   border-radius: 4px;
   border: 1px solid var(--theme-splitter-color);
-  font-family: sans-serif;
+  font-family: system-ui, Arial, sans-serif;
```

Comment:
Let's go ahead and just delete the `font-family` styling altogether for these selectors. That way it conveniently defaults to the system-ui.

Reply:
True!